tests/testDbLocal.py

```diff
@@ -476,6 +477,10 @@ def main():
         datefmt='%m/%d/%Y %I:%M:%S', 
         level=logging.DEBUG)
 
+    if not os.path.isfile(TestDbLocal.CREDFILE):
```

Comment:
This test needs to be done after `expanduser()`, `isfile()` does not work on non-expanded names. 
And this test repeats the test below, so it's not really needed, what is probably needed here:

    # always call expanduser(), no harm if there is no ~ in the name
    credFile = os.path.expanduser(TestDbLocal.CREDFILE)
    if not os.path.isfile(credFile):
        logging.warning("Can't test, %s not found", credFile)
    else:
        unittest.main()